.github/workflows/release.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         run: echo VERSION=${GITHUB_REF_NAME#v} >> $GITHUB_ENV
 
       - name: Generate changelog with git-cliff
-        uses: tj-actions/git-cliff@75599f745633e29f99bd9e14a30865b7d2fcbe84 # v1.5.0
+        uses: tj-actions/git-cliff@7b2afa2d0b650c50cde5a7b173a74753ecd29072 # v2.0.2
         with:
           args: --latest --strip all
```

Comment:
`args` was dropped, so we can't upgrade yet

- https://github.com/tj-actions/git-cliff/issues/80